addons/website_slides/models/slide_question.py

```diff
@@ -45,6 +47,11 @@ def _compute_statistics(self):
             question.attempts_avg = stats.get('attempts_count', 0) / stats.get('attempts_unique', 1) if stats else 0
             question.done_count = stats.get('done_count', 0) if stats else 0
 
+    @api.depends('answer_ids.is_correct')
+    def _compute_answers(self):
```

Comment:
Can you make this method compute a `validation_error` Char field, and use it in your template with < t-if="validation_error"> to avoid repeating both the logic and the alert layout?
This will also have the benefit of being self-explanatory as to why there are even a method and fields at all.